### PR TITLE
Fix clang warnings and replace tabs

### PR DESCRIPTION
**Pull request by MaartenBent**

### CODE REVIEW

File: src/html/chm.cpp [hunk 1]
```diff
@@ -856,6 +856,7 @@ wxFSFile* wxChmFSHandler::OpenFile(wxFileSystem& WXUNUSED(fs),
  */
 wxString wxChmFSHandler::FindFirst(const wxString& spec, int flags)
 {
+    wxUnusedVar(flags);
```
Comment (discnl) [inline src/html/chm.cpp:859]: Looks like this one could use `WXUNUSED` instead as well?

File: src/osx/cocoa/dnd.mm [hunk 2]
```diff
@@ -318,6 +318,9 @@ - (NSDragOperation)code
 
 - (NSDragOperation)draggingSession:(nonnull NSDraggingSession *)session sourceOperationMaskForDraggingContext:(NSDraggingContext)context
 {
+    wxUnusedVar( session );
+    wxUnusedVar( context );
```
Comment (discnl) [inline src/osx/cocoa/dnd.mm:322]: Like with most other changes in this commit already: there should be no spaces around the parameters here (and elsewhere in this commit).

**Review by discnl — COMMENTED** [review]
Thanks for these. Were all relevant tabs supposed to be replaced? There's at least one in `include/wx/msw/uxtheme.h` and `samples/dataview/dataview.cpp` and two in `tests/controls/textctrltest.cpp`.

I had a typo fixed in a line that also contained a tab and was wondering what would then happen with the [whitespace checks](https://github.com/wxWidgets/wxWidgets/runs/1592339406) (which are off again for now it seems). So I was going to replace tabs first, thanks for doing it already!

**Comment by MaartenBent** [comment]
I encountered some tabs when fixing the warnings, and then replaced them in `src/`. I indeed forgot to check the other files and directories. I'll replace those as well.

GitHub actions are not triggered when there are only changes in `src/` (and `build/`).

**Comment by discnl** [comment]
Ah, `code_checks.yml`, thanks.

I thought you might have already checked the repo before because most tabs were gone now besides at least the files mentioned (and `ScintillaWX.cpp` IIRC), (almost?) all other hits were in third-party sources.

**Pull request merged**